tensorboard/webapp/metrics/store/metrics_reducers.ts

```diff
@@ -981,15 +981,28 @@ const reducer = createReducer(
   on(actions.rangeSelectionToggled, (state) => {
     const nextRangeSelectionEnabled = !state.rangeSelectionEnabled;
     let nextStepSelectorEnabled = state.stepSelectorEnabled;
+    let linkedTimeSelection = state.linkedTimeSelection;
 
     if (nextRangeSelectionEnabled) {
       nextStepSelectorEnabled = nextRangeSelectionEnabled;
+      if (!linkedTimeSelection) {
+        linkedTimeSelection = {
+          start: {step: state.stepMinMax.min},
+          end: {step: state.stepMinMax.max},
+        };
```

Comment:
For my own reference, this is because linked time selection will be set [here](https://cs.opensource.google/tensorflow/tensorboard/+/master:tensorboard/webapp/metrics/store/metrics_reducers.ts;l=959?q=tensorboard%2Fwebapp%2Fmetrics%2Fstore%2Fmetrics_reducers.ts&ss=tensorflow%2Ftensorboard) (to single selection) if we don't update it when range is enabled.

The clicking flow to not update here is: enable SS --> enable Range --> enable LT
(then we will find only one fob as the result)

Riley, feel free to add more and correct me.